<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1902</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MergeSortButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>MERGE SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuickSortButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>QUICK SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertionSortButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>INSERTION SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CountingSortButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>COUNTING SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RadixSortButton">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>RADIX SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BubbleSortButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>BUBBLE SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectionSortButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>SELECTION SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CombSortButton">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>420</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>COMB SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BucketSortButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="text">
     <string>BUCKET SORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ImportButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import Data From CSV</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>1381</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>312</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>851</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="StartStopLayout">
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PAUSE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RESUME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="SelectingColumnsLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Columns for Sorting:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectingColumnsComboBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>627</y>
      <width>771</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="SearchingColumnLayout">
     <item>
      <widget class="QLineEdit" name="column1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column4"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column5"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column6"/>
     </item>
     <item>
      <widget class="QLineEdit" name="column7"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="SearchingLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>630</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Searching Data By Columns:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>700</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectingColumnsComboBox_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>360</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>770</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>810</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Starts with</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ends with</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Contains</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>760</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AND</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOT</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1902</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
